tests/test_mdns_service.js

```diff
@@ -0,0 +1,29 @@
+var dns_sd = require('../lib/dns_sd')
+  , proxyquire =  require('proxyquire')
```

Comment:
Hehe... apparently I have no idea what's going on in my software. So, thanks for the explanation.